src/traits.rs

```diff
@@ -1486,6 +1486,11 @@ pub trait ComponentExt: Debug {
     ///     println!("{}°C", component.temperature());
     /// }
     /// ```
+    ///
+    /// ## Linux hwmon
```

Comment:
```suggestion
    /// ## Linux
```

No need to add `hwmon`, it's more of an internal thing. (Same below)